lerna.json

```
@@ -2,7 +2,6 @@
   "version": "independent",
   "npmClient": "npm",
   "packages": [
-    "api",
     "packages/*",
```

Looks like this went missing in the process. :slightly_smiling_face: 

```suggestion
    "api",
    "packages/*",
```